src/config.js

```diff
@@ -396,11 +396,16 @@ export default class Config {
     }
 
     if (process.platform === 'win32') {
-      if (this.plugnplayEnabled) {
-        this.reporter.warn(this.reporter.lang('plugnplayWindowsSupport'));
+      const cacheRootFolderDrive = path.parse(this._cacheRootFolder).root;
+      const lockfileFolderDrive = path.parse(this.lockfileFolder).root;
+
+      if (cacheRootFolderDrive !== lockfileFolderDrive) {
```

Comment:
There should be case-insensitive check.

Reply:
Makes sense! Can you open a PR?

Reply:
It looks like both `cacheRootFolder` and `lockfileFolder` can be user customized. Would it make more sense for user-provided values to be normalized at time of ingestion to avoid these kinds of one-off normalizations in other places in the code as well?